lib/util.ts

```diff
@@ -6,64 +6,107 @@ import {LIB_VERSION} from './version'
 
 export function sleep(timeMs: number): Promise<void> {
     return new Promise(resolve => {
+        if (!Number.isFinite(timeMs)) {
```

Comment:
bit of a random change - this is only used a few places in the lib itself, none of which take `Infinity` afaik